### ISSUE TITLE
Config for branches to ignore

### ISSUE THREAD

**Issue by drpayyne** [bk201-/jira-prepare-commit-msg#438]
Can we have a config for a list of branches to ignore?

Either a pattern, or an array. For example, I don't want this script to run when I'm on either of these branches - `master` `develop`.

Basically, I don't want an error to display like below when I'm on a branch without a JIRA ticket ID like `develop`.

```
JIRA prepare commit msg > start
JIRA prepare commit msg > Error: The JIRA ticket ID not found
JIRA prepare commit msg > done
✨  Done in 0.12s.
```

**Comment by jsg2021** [comment]
while that works, it spreads the config around. It would be nice to keep it consolidated.

**Comment by bk201-** [comment]
Hi! I've fixed it in 1.7.0. 

Now you can use the new settings flag:
- ignoredBranchesPattern. By default it is `^(master|main|dev|develop|development|release)$`
- ignoreBranchesMissingTickets. Set it to `true` to ignore all branches without a JIRA ticket
